java/server/src/org/openqa/selenium/docker/Container.java

```diff
@@ -35,7 +35,7 @@
   private final ContainerId id;
 
   public Container(Function<HttpRequest, HttpResponse> client, ContainerId id) {
-    LOG.info("Created container " + id);
+    LOG.finest("Created container " + id);
```

Comment:
"This code is new and not tested well. While we may drop the log level before we ship 4.0, right now this is extremely helpful to users."